src/main/docker/bamboo/swift/Dockerfile

```diff
@@ -17,28 +17,13 @@ RUN update-alternatives --set java /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/jav
 # Add file in /artemis/bin/mvn that uses the correct java version and passes all arguments to mvn
 RUN mkdir /artemis && mkdir /artemis/bin && printf '#!/bin/bash\nJAVA_HOME=/usr/lib/jvm/java-15-oracle /usr/bin/mvn "$@"\n' > /artemis/bin/mvn && chmod 777 /artemis/bin/mvn
 
-# Install Swift and SwiftLint (Adapted from norionomura/swift and norionomura/swiftlint)
-ENV SWIFT_BRANCH swift-5.3.1-release
-ENV SWIFT_PLATFORM ubuntu18.04
-ENV SWIFT_VERSION 5.3.1-RELEASE
+# Install Swift by copying relevant files
+COPY --from=swift:latest /usr /usr
```

Comment:
this is standard, see: https://docs.docker.com/develop/develop-images/multistage-build/#use-an-external-image-as-a-stage